src/NUnitEngine/nunit.engine/Internal/NUnitConfiguration.cs

```diff
@@ -59,35 +59,28 @@ public static string MonoExePath
         {
             get
```

Comment:
I'm slightly worried about the "shouldn't make a difference" part. Have you verified that it still works correctly? I can see that the code "should" do the same thing, but if I made these changes myself, I would not trust it without seeing it work. Since this is a level of functionality beyond unit testing, you would have to run it on a Windows machine with mono installed and either step through or look at the test output. We do have a test that lists the available runtimes to the console, so that might do it for you.

This is a good example of where even a small change would benefit from being done in a separate PR, but let's try to keep this in if we can. If you don't have a machine set up to test it, I can do it.